spec/models/user_spec.rb

```diff
@@ -19,7 +18,25 @@
     expect(user).to_not be_valid
   end
 
+  context 'on update' do
+    before { user.save! }
+
+    context 'without skills' do
+      before { user.skill_list.add([]) }
+    
+      it { is_expected.to_not be_valid }
+    end
+
+    context 'with skills' do
+      before { user.skill_list.add(['Analytics']) }
+    
+      it { is_expected.to be_valid }
+    end
+  end
+
   describe 'Complete profile' do
```

Comment:
Is this description doesn't tell me much about what is being tested iwthin it? The tests contained within it test particular methods on the `User` model - so shouldn't we split up the tests, each with its own describe block like so:

```
describe '#has_complete_profile?' do
...#tests
end

describe '#has_correct_skills??' do
...#tests
end

Reply:
if you want to improve legibility, please just feel free to do it!